### PR TITLE
(394) Suppress logs when testing

### PR DESCRIPTION
**Pull request by yndajas**
## Context

When running tests locally, we were getting a lot of noise in the console from the logger, making it hard to parse the overall picture, and - most importantly - hard to spot the errors and their stack traces

## Changes in this PR

This suppresses all but fatal logging, which in practice means we just see the standard Jest and Cypress output plus stack traces of any errors

##### Before

image

##### After

image

##### Before

image

image

##### After

image

##### Before

image

##### After

image

##### Before

image

##### After

image

### CODE REVIEW

**Comment by yndajas** [comment]
[Stop warning messages being logged in tests](https://trello.com/c/ruTPpPD6/394-stop-warning-messages-being-logged-in-tests)

**Review by Gweaton — APPROVED** [review]
